docs/tutorials/RemovingFields.md

```diff
@@ -28,7 +28,8 @@ IntelliJ IDEA provides a refactoring tool that can identify *most* parts of a re
 
 ### Assisted refactoring
 
-The `address` field in `Person` is actually an instance of the `seedu.address.model.person.Address` class. Since removing the `Address` class will break the application, we start by identifying `Address`'s usages. This allows us to see code that depends on `Address` to function properly and edit them on a case-by-case basis. Right-click the `Address` class and select `Refactor` \> `Safe Delete` through the menu.
+The `address` field in `Person` is actually an instance of the `ezschedule.model.person.Address` class.
```

Comment:
I think `Person` class and `address` field is not longer applicable in our case, but this can be fixed later.